docs/release-notes/2.14.0.md

```diff
@@ -0,0 +1,79 @@
+
+- DBaaS Preview phase 1.0 (Technical  Preview)
+
+    DBaaS enables non-DBAs (software architects, developers, site reliability engineers, etc.) to perform typical DBA tasks to manage an organization's database environment via user interfaces and automation orchestration.
+
+    **Note**: This capability is feature-flagged and turned off by default. Users require a variable to be passed to PMM to expose this functionality.
+
+- Pass configured SSL certificates to connect PMM for MongoDB
+
+    Enables PMM administrators to setup configured SSL certificate keys to authenticate the connection to PMM, specifically for setting up MongoDB. This is a critical security requirement especially in large enterprise infrastructure environments.
+
+- External services presentation on node summary dashboard
+
+    Improvements to the user experience for adding and viewing external services (any data that can be monitored by a Prometheus exporter such as: non-Percona supported databases like Reddis, ElasticSearch, Cassandra, etc. or an organization's external application) on the Node Summary dashboard of PMM.
+
+
+## New Features
+
+* [PMM-5765](https://jira.percona.com/browse/PMM-5765): Add external serverless service (no host or address) with new 'pmm-admin add external-serverless' option
+* [PMM-7015](https://jira.percona.com/browse/PMM-7015): DBaaS Preview: Create DB cluster with randomly-generated password
+* [PMM-6941](https://jira.percona.com/browse/PMM-6941): Integrated Alerting: Ability to delete an 'Alert Rule Template'
+
+
+
+## Improvements
+
+* [PMM-7145](https://jira.percona.com/browse/PMM-7145): 'Push' metrics mode is default when adding services and nodes (All agents collecting data from Services and Nodes will now use PUSH model if not specified explicitly. You will still be able to use '-metrics-mode' flag to use Pull metrics if needed. All previously set up agents will keep their existing mode. To change these you need to remove and re-add them.)
+* [PMM-7282](https://jira.percona.com/browse/PMM-7282): Integrated Alerting: Ability to create rule without channels and filters
```

Comment:
PMM-7282 - is from the feature in Technical Preview (Integrated Alerting) . should we group them and mark them once?

Reply:
Jirabot can't sort on that (unless you add a flag and adjust the code)